5.5/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM centos:centos7
+
+# MySQL image for OpenShift.
+#
+# Volumes:
+#  * /var/lib/mysql - Datastore for MySQL
+# Environment:
+#  * $MYSQL_ROOT_PASSWORD - Password for MySQL root user
+#  * $MYSQL_USER - User to create in first invocation
+#  * $MYSQL_PASSWORD - User's password
+#  * $MYSQL_DATABASE - Name of the database to create
+
+MAINTAINER  Martin Nagy <mnagy@redhat.com>
+EXPOSE 3306
+
+RUN groupadd -g 27 -o -r mysql && \
+    useradd -M -N -g mysql -o -r -d /var/lib/mysql -s /sbin/nologin -c "MySQL Server" -u 27 mysql
+
+RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
+    yum -y --setopt=tsflags=nodocs install https://www.softwarecollections.org/en/scls/rhscl/mysql55/epel-7-x86_64/download/rhscl-mysql55-epel-7-x86_64.noarch.rpm && \
+    yum -y --setopt=tsflags=nodocs install hostname mysql55 && \
+    yum clean all
+
+ADD mysql /opt/openshift/
```

Comment:
we use COPY for this in other images, maybe be consistent? https://github.com/mfojtik/sti-nodejs/blob/metadata/0.10/Dockerfile#L20